wgsl/index.bs

```diff
@@ -2277,6 +2277,27 @@ Details of conversion to and from floating point are explained in [[#floating-po
   <thead>
     <tr><th>Precondition<th>Conclusion<th>Notes
   </thead>
+  <tr algorithm="vector coercion of unsigned integer to boolean">
+     <td>|e| : vec|N|&lt;u32&gt;
+     <td>`vec`|N|&lt;`bool`&gt;`(`|e|`)` : vec|N|&lt;bool&gt
+     <td>Component-wise coercion of a unsigned integer vector to a boolean vector.<br>
+         Component |i| of the result is `bool(`|e|`[`|i|`])`<br>
+         (OpINotEqual to compare |e| against a zero vector.)
```

Comment:
I don't think this line is right given we're doing component wise checks?

Reply:
OpINotEqual operates on vector operands, producing a vector result of the same component count, and "Results are computed per component."